shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/exec/context/CommonBeans.java

```diff
@@ -72,10 +73,16 @@
   private static final Logger LOG = LoggerFactory.getLogger(CommonBeans.class);
 
   @Bean
-  Configuration baseConfiguration(@Value("${instance.workspace}") String workspace) {
+  Configuration baseConfiguration(
+      @Value("${instance.workspace}") String workspace,
+      @Value("${ct-config:#{null}}") String circusTrainConfigLocation) {
```

Comment:
`@Value("${ct-config:#{null}}")` is the syntax for getting null value if `ct-config` is not provided.

Spring throws:

`java.lang.IllegalArgumentException: Could not resolve placeholder 'ct-config' in value "${ct-config}"`

if we don't set the value as null explicitly.